### DISCUSSION TITLE
Does gopro video file keep qrcode settings applied in video metadata?

### DISCUSSION THREAD

**Discussion by carbendi** [gopro/labs#871] [Q&A] [answered]
I was wonderin if gopro video file keeps the qrcodes applied maybe somewhere in the metadata. Basically I shot something long ago with a custom log curve and I want to mimic that again but don't remember which calues I used. 

Would it be possible to extract that from metadata?

**Comment by dnewman-gpsw** [comment, marked as answer]
Yes it does. Extract metadata using https://gopro.github.io/labs/control/metadata/  Labs metadata is in User Metadata

**Reply by carbendi** [reply]
ah, thank you this helped greatly!
